### PR TITLE
Feature/thirdparty enricher

### PR DESCRIPTION
**Pull request by afek854**

### CODE REVIEW

**Comment by afek854** [comment]
Blocked until IG

**Comment by afek854** [comment]
We're using inspektor gadget by commit hash until new release

File: pkg/containerwatcher/v1/container_watcher.go [hunk 1]
```diff
@@ -209,12 +210,13 @@ func CreateIGContainerWatcher(cfg config.Config, applicationProfileManager appli
 		if len(event.Args) > 0 {
 			path = event.Args[0]
 		}
+
+		ruleManager.ReportEvent(utils.ExecveEventType, &event)
```
[the remark this answers is not in the document]
Comment (afek854) [reply pkg/containerwatcher/v1/container_watcher.go:214]: It does not matter

File: pkg/ebpf/gadgets/hardlink/tracer/hardlink_bpfel.go [hunk 2]
```diff
@@ -24,7 +25,7 @@ type hardlinkEvent struct {
 	Exepath    [4096]uint8
 	Oldpath    [4096]uint8
 	Newpath    [4096]uint8
-	_          [7]byte
+	_          [3]byte
```
Comment (matthyx) [inline pkg/ebpf/gadgets/hardlink/tracer/hardlink_bpfel.go:28]: padding?

File: pkg/malwaremanager/v1/malware_manager.go [hunk 3]
```diff
@@ -146,19 +147,19 @@ func (mm *MalwareManager) getWorkloadIdentifier(podNamespace, podName string) (s
...
+		mm.reportFileExec(&exec.Event)
 	case utils.OpenEventType:
-		open, ok := event.(*traceropentype.Event)
+		open, ok := event.(*events.OpenEvent)
 		if !ok {
 			logger.L().Error("MalwareManager - failed to cast event to open event")
 			return
 		}
-		mm.reportFileOpen(open)
+		mm.reportFileOpen(&open.Event)
```
Comment (matthyx) [inline pkg/malwaremanager/v1/malware_manager.go:162]: do you really have to dereference here?

**Review by matthyx — APPROVED** [review]

**Review by amitschendel — CHANGES_REQUESTED** [review]

**Pull request merged**